packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +126,31 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  
+  all: function() {
+    this.allDeps.depend();
+    var ret = {};
+    _.each(this.keys, function(value, key) {
+      ret[key] = parse(value);
+    });
+    return ret;
+  },
+  
+  clear: function() {
+    var self = this;
+    
+    var oldKeys = self.keys;
+    self.keys = {};
+    
+    self.allDeps.changed();
+    
+    _.each(oldKeys, function(value, key) {
```

Comment:
Oh, I see. I think I would have `anyDeps` and `allDeps` separately maybe?

- `allDeps` is for the situation where all of the keys change - everything should listen to it
- `anyDeps` is triggered on any change, and only `all` depends on it